simphony/testing/abc_check_mesh.py

```diff
@@ -384,11 +400,14 @@ def test_update_item_points(self):
         container = self.container
         uids = self._add_items(container)
         item = self.get_operation(container, uids[2])
+        point_uids = [
+            container.add_point(Point((1.0 * i, 1.0 * i, 1.0 * i)))
```

Comment:
> The points added at the setup are used in all the tests which in general relay on a specific number of points available.

I might still be missing something or incorrectly looking at things, but I don't see where the points are added to the mesh-container though.  Besides ```test_update_item_points```, the other tests (e.g. test_adding_and_getting_items for MeshEdgeOperationsCheck)  occur with a container without any points.